tests/QueueTest.php

```diff
@@ -377,10 +369,6 @@ public function testGetJobs()
 
     public function testHugeJob()
     {
-        // Default value - maxBatchSize = 1000
-        setConfig('algoliasearch/queue/number_of_job_to_run', 10);
```

Comment:
Because they settings the default value, which are set in `setUp()` method. I might have missed something though ^^